src/main/java/org/osiam/resources/controller/UserController.java

```diff
@@ -61,7 +67,8 @@
     @Autowired
     private TokenService tokenService;
 
-    private AttributesRemovalHelper attributesRemovalHelper = new AttributesRemovalHelper();
+    @Autowired
+    private AttributesRemovalHelper attributesRemovalHelper;
```

Comment:
Not with this PR. I will create an own PR for complete switch to constructor injection.